lib/msf/core/modules/external/bridge.rb

```diff
@@ -36,12 +36,16 @@ def get_status
   def initialize(module_path)
     self.running = false
     self.path = module_path
+    self.env = {
+      'PYTHONPATH' => ENV['PYTHONPATH'] + File::PATH_SEPARATOR + File.expand_path('../python', __FILE__)
```

Comment:
Adding `env` to the class is a good idea, but I think adding this line would be a better fit in a class that looked like:
```ruby
class Msf::Core::Modules::External::PyBridge < Msf::Core::Modules::External::Bridge
  def self.applies?(module_name)
    module_name.match? /\.py$/
  end

  def initialize(module_path)
    super
    self.env = self.env.merge {'PYTHONPATH' => ENV['PYTHONPATH'] + File::PATH_SEPARATOR + File.expand_path('../python', __FILE__)}
  end
end
```

And adding `Bridge.open` that auto-chooses the most applicable class. This approach will later enable us to add bits that will construct the correct invocation on Windows, or even use compiled languages (would require a compiler, though).